src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Nullable.cs

```diff
@@ -2565,6 +2565,91 @@ public class Program
             EmitPrivateMetadata(source, expectedPublicOnly, expectedPublicAndInternal, expectedAll);
         }
 
+        [Fact]
+        [WorkItem(37161, "https://github.com/dotnet/roslyn/issues/37161")]
+        public void EmitPrivateMetadata_ExplicitImplementation()
```

Comment:
Is this add just a weird merge or something? Doesn't seem relevant.

Reply:
An unrelated test but intentional.

---
In reply to: [304055713](https://github.com/dotnet/roslyn/pull/37271#discussion_r304055713) [](ancestors = 304055713)